### PR TITLE
update pymongo to 3.6.0

### PR DESCRIPTION
**Pull request by enlighter**

### CODE REVIEW

**Comment by enlighter** [comment]
Auto build check is giving this error: "Error: bad character '=' in package name dependency 'dnspython>=1.8.0,<2.0.0' ". How to set the version constraints here, the way i have put it works in conda.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,6 +21,10 @@ requirements:
     - setuptools
   run:
     - python
+    - pykerberos
+    - dnspython>=1.8.0,<2.0.0
```
Comment (ericdill) [inline recipe/meta.yaml]: this needs a space between ``dnspython`` and ``>=1.8.0,<2.0.0``

**Comment by ericdill** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
The re-rendering is failing because of the formatting issue that you have pointed out in the spec.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Pushed a commit to fix this.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,6 +21,10 @@ requirements:
     - setuptools
   run:
     - python
+    - pykerberos
```
Comment (jakirkham) [inline recipe/meta.yaml]: Requires PR ( https://github.com/conda-forge/staged-recipes/pull/2701 ). Any interest in taking that over?

**Pull request closed without merging**